src/intro/hello-world.md

```diff
@@ -4,104 +4,203 @@
+your project**. You can learn more about Godot's resource paths [here][ref-resource-paths].
+```
+
+
+#### Other possible target paths
+
+The following is a list of other possible paths depending on the target of your library. You can copy them as-is for this example, replacing
+`{myCrate}` with the name of your actual library:
+
+```ini
+[configuration] # same as before
+entry_symbol = "gdext_rust_init"
+compatibility_minimum = 4.1
+
+[libraries]
+linux.debug.x86_64 =     "res://../{myCrate}/target/debug/lib{myCrate}.so"
+linux.release.x86_64 =   "res://../{myCrate}/target/release/lib{myCrate}.so"
+windows.debug.x86_64 =   "res://../{myCrate}/target/debug/{myCrate}.dll"
+windows.release.x86_64 = "res://../{myCrate}/target/release/{myCrate}.dll"
+macos.debug =            "res://../{myCrate}/target/debug/lib{myCrate}.dylib"
+macos.release =          "res://../{myCrate}/target/release/lib{myCrate}.dylib"
+macos.debug.arm64 =      "res://../{myCrate}/target/debug/lib{myCrate}.dylib"
+macos.release.arm64 =    "res://../{myCrate}/target/release/lib{myCrate}.dylib"
+```
+
+The `..` is a pointer to a parent folder of the end path.
+Which means Godot will look for the library around the parent folder of the project for its path.
+
+```admonish tip
+You can also employ the use of synbolic links and git modules and then treat those as regular folders and files. Godot reads those just fine too! 
 ```
 
 ```admonish note
-If you specify your cargo compilation target via the `--target` flag or a `.cargo/config.toml` file, the rust library will be placed in a 
-path name that includes target architecture, and the `.gdextension` library paths will need to match. For example, for M1 Macs 
+If you specify your cargo compilation target via the `--target` flag or a `.cargo/config.toml` file, the rust library will be placed in a path name
+that includes target architecture, and the `.gdextension` library paths will need to match. For example, for M1 Macs 
 (`macos.debug.arm64` and `macos.release.arm64`), the path would be `"res://../rust/target/aarch64-apple-darwin/debug/lib{myCrate}.dylib"`.
 ```
 
 
 ### `extension_list.cfg`
 
-A second file `res://.godot/extension_list.cfg` should be generated once you open the Godot editor for the first time.
-If not, you can also manually create it, simply containing the Godot path to your `.gdextension` file:
+A second file `res://.godot/extension_list.cfg` should be generated once you open the Godot editor for the first time. If not, you can also manually
+create it, simply containing the Godot path to your `.gdextension` file:
 
 ```text
 res://HelloWorld.gdextension
 ```
 
 
-## Cargo project
+---
 
-In your terminal, create a new Cargo project with your chosen crate name:
+From here on out, your Rust project files should be connected to Godot, so now we'll make it actually function.
```

Comment:
The line is not needed.

"actually function" is again imprecise. What is left?
It's also fine to move this to the next section without announcing it beforehand.